modern/src/App.js

```diff
@@ -177,6 +224,9 @@ function Layer({ node, id, image, children, x, y }) {
   if (img.h !== undefined) {
     params.height = Number(img.h);
   }
+  if (id === "volumethumb") {
+    params.border = "1px solid black";
+  }
```

Comment:
Still trying to figure out why this image isn't properly loading. Leaving this in for now in case you want to test it